lib/experimental/disable-tinymce.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Experiment to disable TinyMCE and the Classic block.
+ *
+ * @package gutenberg
+ */
+
+// add_action( 'admin_footer', 'gutenberg_test_tinymce_access' ); // Uncomment the following line to force an external TinyMCE usage.
```

Comment:
I'm going to keep that for now, just in case we'd like to test 3rd party global usage.